### DISCUSSION TITLE
Please add a LICENSE file

### DISCUSSION THREAD

**Discussion by gmacario** [space-ros/space-ros#280] [Ideas]
See https://github.com/ros2/ros2/issues/1590

**Comment by asimonov** [comment]
interesting point. 

for ros2 repo's issue the comment is that ros2 is meta-information repository, there is no code in it, so there is nothing for license to cover.

for space-ros repo we have some docker/build/CI code, so may be good idea to add a license

**Comment by gmacario** [comment]
> for space-ros repo we have some docker/build/CI code, so may be good idea to add a license

Based on https://github.com/ros2/ros2/issues/1590#issuecomment-2307499711 I concur.

**Comment by gmacario** [comment]
Hello space-ros maintainers,

> > for space-ros repo we have some docker/build/CI code, so may be good idea to add a license
> 
> Based on [ros2#1590 (comment)](https://github.com/ros2/ros2/issues/1590#issuecomment-2307499711) I concur.

Shall @mkhansenbot comment at https://github.com/space-ros/demos/issues/29#issuecomment-2274125338

> Everything here is Apache 2 by default, but I'll add a LICENSE file. Sorry for any confusion

apply to this repository as well?

If so and nobody else wants to, I am happy to contribute the corresponding PR here.

This fork of ros/ros2 has already some code (i.e under `docker` and `.github/workflows`) and IMHO it is time to make things clear, because  according to https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/licensing-a-repository

> ... without a license, the default copyright laws apply, meaning that you retain all rights to your source code and no one may reproduce, distribute, or create derivative works from your work. If you're creating an open source project, we strongly encourage you to include an open source license.

**Comment by ivanperez-keera** [comment]
I would be ok with this. @gmacario can you send a PR adding a license at the top level, as well as notices to any scripts or code as needed?
